src/Twilio/Http/Request.cs

```diff
@@ -24,7 +24,7 @@ public class Request
         /// </summary>
         public HttpMethod Method { get; }
 
-        private Uri Uri;
+        public Uri Uri;
```

Comment:
why this change?

Reply:
In order to log the request Uri in the _TwilioRestClient_ the Uri needs to be public.

Reply:
Then you should add the `{ get; private set; }` [like QueryParams](https://github.com/twilio/twilio-csharp/pull/545/files#diff-892488c88ebf07d450c8a3a13ccafcc08dfe907d422c5e9001f6aedd21b78718R52)